data/dc/dc_limits copy.geojson

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Should this file be here? It is copied from the other dc_limits file. Keep just one or comment why there must be two.

Reply:
No these files shouldn't be there they don't show up in my repository so I don't know why they're in the pull request there should just be one